test/unit/mdc-dialog/util.test.js

```diff
@@ -24,21 +24,11 @@
 import bel from 'bel';
 import td from 'testdouble';
 import {assert} from 'chai';
-
 import * as util from '../../../packages/mdc-dialog/util';
 
 suite('MDCDialog - util');
 
-// Babel transpiles optional function arguments into `if` statements. Istanbul (our code coverage tool) then reports the
-// transpiled `else` branch as lacking coverage, but the coverage report UI doesn't tell you where the missing branches
-// are. See https://github.com/gotwarlost/istanbul/issues/582#issuecomment-334683612.
-// createFocusTrapInstance() has two optional arguments, so code coverage reports two missed branches.
-test('createFocusTrapInstance covers `if` branches added by Babel transpilation of optional arguments', () => {
-  const surface = bel`<div></div>`;
-  util.createFocusTrapInstance(surface);
-});
-
-test('createFocusTrapInstance creates a properly configured focus trap instance', () => {
+test('#createFocusTrapInstance creates a properly configured focus trap instance with all args specified', () => {
```

Comment:
Again wondering why dialog changes are in a drawer PR?

Also, nit: dialog/util is not a class, it's just a set of functions, so `#` is inaccurate (typically denotes methods)

Reply:
I noticed inconsistencies between Dialog and Drawer and figured I would fix them while I'm here.

Removed `#` from all dialog and drawer util test descriptions.